mesa/space.py

```diff
@@ -4,9 +4,12 @@
 
 Objects used to add a spatial component to a model.
 
-Grid: base grid, a simple list-of-lists.
-SingleGrid: grid which strictly enforces one object per cell.
-MultiGrid: extension to Grid where each cell is a set of objects.
+Grid: base grid, it creates a square grid.
```

Comment:
Should be rectangular. I know the `Grid` docstring says square, which should be fixed as well.

Reply:
right, I blindly copy-pasted x)

Reply:
done, fixed also the docstring in the Grid class